### PR TITLE
Add RabbitMQ, Redis, Memcached

### PR DESCRIPTION
**Pull request by andreiavrammsd**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,3 +101,57 @@ services:
     volumes:
       - ./consul.json:/config/consul.json
 ```
+
+- RabbitMQ
+```yaml
+version: '3.4'
```
Comment (cristaloleg) [inline README.md]: Please, drop version here and below 😉

File: README.md [hunk 2]
```diff
@@ -101,3 +101,57 @@ services:
...
+
+services:
+  rabbitmq:
+    image: rabbitmq:3.7-management-alpine
+    restart: unless-stopped
+    ports:
+      - "5672:5672"
+      - "15672:15672"
```
Comment (cristaloleg) [inline README.md]: Care to add a comment? AFAIR it's queue and management respectively.

File: README.md [hunk 3]
```diff
@@ -101,3 +101,57 @@ services:
...
+
+services:
+  redis:
+    image: redis:5-alpine
+    restart: unless-stopped
+    ports:
+      - "6379:6379"
+    volumes:
+      - redis:/data
+    command: redis-server --appendonly yes
```
Comment (cristaloleg) [inline README.md:137]: Hm, do we need this? I thought it should start without any additional commands 🤔

**Review by cristaloleg — APPROVED** [review]
Huh, thank you for the contribution, I have no idea how you found it, but open source is amazing 🎉 
Care to fix few comments? BTW, looks like in some places there are 2 spaces and in some 4, can you check it?

File: README.md (hunk 3 above)
Comment (andreiavrammsd) [reply README.md:137]: It's for data persistence, in case of server restart.

Comment (cristaloleg) [reply README.md:137]: Ah, got it.

**Comment by cristaloleg** [comment]
Thank you @andreiavrammsd

**Pull request merged**